libs/initScripts/prod-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@hareeshnagaraj we wouldn't be committing this file

Reply:
i actually really like this - would it be a ton of effort to do the same with the local configs? also we can commit without the values filled in safely

Reply:
@hareeshnagaraj for local the values are just from contract-config.json for both data and eth contracts